pkgs/applications/office/pympress/default.nix

```
@@ -29,7 +30,16 @@ python3Packages.buildPythonApplication rec {
   buildInputs = [
     gtk3
     poppler_gi
-  ] ++ lib.optional withGstreamer libcanberra-gtk3;
+  ] ++ lib.optionals withGstreamer [
+    libcanberra-gtk3
+    gst_all_1.gstreamer
+    gst_all_1.gst-plugins-base
+    gst_all_1.gst-plugins-bad
+    gst_all_1.gst-plugins-ugly
+    (gst_all_1.gst-plugins-good.override {gtkSupport = true;})
+    gst_all_1.gst-libav
+    gst_all_1.gst-vaapi
```

```suggestion
    gst_all_1.gstreamer
    gst_all_1.gst-plugins-base
    gst_all_1.gst-plugins-bad
    (gst_all_1.gst-plugins-good.override {gtkSupport = true;})
```
That's what I take from the upstream docs https://github.com/Cimbali/pympress/blob/fc5ebfc79b5255ea801e4935aaa33f7bcadc595d/README.md#media-and-autoplay

I am not sure if the others are even used or loaded on demand.

---

I think the video playback is completely handled by gstreamer so I would assume the instance of gstreamer is independent from the rest of the package and can access all available plugins/extensions.
In the upstream docs it says: "and plugins gstreamer-good/-bad/-ugly based on which codecs you need" so why remove `gst_all_1.gst-plugins-ugly`?
I have taken the example from the wiki https://nixos.wiki/wiki/GStreamer for full gstreamer support.